### ISSUE TITLE
Add Expresion and Attribute support in read/write params

### ISSUE THREAD

**Issue by cerveada** [AbsaOSS/spline#973]
Since Spark Agent 0.6.3 Expressions and Attributes can be present in read and write operations.

This doesn't cause any issues, but UI doesn't show all available information and/or shows them as raw data.

**Comment by wajda** [comment]
No, sorry. It's actually a UI issue. Moving it back to the UI repo :)
